[1.16.5] No enum constant for Vanilla Biomes!

<!-- ISSUE_TEMPLATE_3 -> IMPORTANT: DO NOT DELETE THIS LINE.-->

<!-- Thank you for reporting ! Please note that issues can take a lot of time to be fixed and there is no eta.-->

<!-- If you don't know where to upload your logs and crash reports, you can use these websites : -->
<!-- https://gist.github.com (recommended) -->
<!-- https://mclo.gs -->
<!-- https://haste.mohistmc.com -->
<!-- https://pastebin.com -->

<!-- TO FILL THIS TEMPLATE, YOU NEED TO REPLACE THE {} BY WHAT YOU WANT -->

**Minecraft Version :** **1.16.5**

**Mohist Version :** **mohist-1.16.5-1099**

**Operating System :** any

**Description of issue :** 

The code

```java
player.getLocation().getBlock().getBiome()
```

Will cause:

```java
Caused by: java.lang.IllegalArgumentException: No enum constant org.bukkit.block.Biome.MINECRAFT_TAIGA
        at java.lang.Enum.valueOf(Unknown Source) ~[?:1.8.0_333]
        at org.bukkit.block.Biome.valueOf(Biome.java:11) ~[forge:?]
        at org.bukkit.craftbukkit.v1_16_R3.block.CraftBlock.biomeBaseToBiome(CraftBlock.java:522) ~[forge:?]
        at org.bukkit.craftbukkit.v1_16_R3.CraftWorld.getBiome(CraftWorld.java:935) ~[forge:7e29f765-296df566-9fb885e8]
        at org.bukkit.craftbukkit.v1_16_R3.block.CraftBlock.getBiome(CraftBlock.java:508) ~[forge:?]
```

Because there is no enum for `Biome.MINECRAFT_TAIGA`, only for `Biome.TAIGA`


So, if the biome is vanilla there is no need for the biome prefix.

--- a/src/main/java/org/bukkit/craftbukkit/v1_16_R3/block/CraftBlock.java
+++ b/src/main/java/org/bukkit/craftbukkit/v1_16_R3/block/CraftBlock.java
@@ -519,8 +519,12 @@ public class CraftBlock implements Block {
         }
 
         Biome biome = org.bukkit.Registry.BIOME.get(CraftNamespacedKey.fromMinecraft(registry.getKey(base)));
-        Biome biome1 =Biome.valueOf(normalizeName(base.getRegistryName().toString()));
-        return (biome == null) ? biome1 : biome;
+
+        if (biome != null){
+            return biome;
+        }
+
+        return Biome.valueOf(normalizeName(base.getRegistryName().toString()));
     }
 
     public static net.minecraft.world.biome.Biome biomeToBiomeBase(Registry<net.minecraft.world.biome.Biome> registry, Biome bio) {

